src/libraries/System.Linq.Expressions/tests/TrimmingTests/ExpressionPropertyTests.cs

```diff
@@ -0,0 +1,79 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+
+/// <summary>
+/// Tests that the System.Linq.Expressions.Expression.Property will correctly preserve both
+/// accessors even if only one is referenced by MethodInfo.
+/// </summary>
+internal class Program
+{
+    static int Main(string[] args)
+    {
+        int result = ExplicitCreation.Test();
+        if (result != 100)
+            return result;
+
+        result = LambdaCreation.Test();
+
+        Type.GetType("foo");
```

Comment:
??

Reply:
I was trying to turn on the warnings - and this is an easy way to get a warning... but I left it in.